grails-app/conf/logback.xml

```diff
@@ -102,7 +99,7 @@
     <logger name="org.hibernate.stat" level="info" />
     <logger name="org.hibernate.transform" level="info" />
     <logger name="org.hibernate.tool" level="info" />
-    <logger name="org.hibernate.type" level="info" />
+    <logger name="org.hibernate.type" level="info" /><!-- https://www.javacodegeeks.com/2012/10/logging-hibernate-sql.html -->
```

Comment:
Would you mind adding a comment to explain the blog post? Are you including to explain why it's used i.e. that it's to improve SQL logging?

Reply:
Yeah exactly. I'll add a comment to clarify